konig-schemagen-maven-plugin/src/main/java/io/konig/maven/KonigSchemagenMojo.java

```diff
@@ -358,6 +362,7 @@ public void execute() throws MojoExecutionException   {
 			
 			computeSizeEstimates();
 			generateViewShape();
+			generateTableShape();
```

Comment:
Replace `generateViewShape()` and `generateTableShape()` with a single method named `generateTabularShapes()`.  

The `generateTabularShapes()` method should invoke the `generateTabularShapes` method on a
new class named `TabularShapeGenerator`.  Pseudo code for this method is shown below.

```
   public void generateTabularShapes(File shapesDir, TabularShapeGeneratorConfig config),
      throws TabularShapeGenerationException {
      /*
         Initialize a TableShapeGenerator and a ViewShapeGenerator with the given config.
        
         For each SQL file found in config.sqlFiles, do:
            Parse the file contents into a net.sf.jsqlparser.statement.Statement
            If the statement is a CreateView statement, invoke the ViewShapeGenerator
            Else if the statement is a CreateTable statement, invoke the TableShapeGenerator
        End Loop

    */
   }
```